packages/notification/src/vaadin-notification.js

```diff
@@ -404,7 +404,11 @@ class Notification extends OverlayClassMixin(ThemePropertyMixin(ElementMixin(Pol
   /** @protected */
   disconnectedCallback() {
     super.disconnectedCallback();
-    this.opened = false;
+    queueMicrotask(() => {
+      if (!this.isConnected) {
```

Comment:
This check doesn't currently get tested. Add `await aTimeout(0);` to the new test also to fix it.

Reply:
Done.